db/seeds.rb

```diff
@@ -17,9 +17,12 @@
 
 # Create default admin user
 admin = User.new { |user|
-  user.name = "admin"
+  user.username = "admin"
   user.full_name = "SSID Administrator"
   user.password_digest = BCrypt::Password.create('$$SSIDPassword$$')
   user.is_admin = true
+  user.last_login_time = Time.now
+  user.email = 'testemail@gmail.com'
```

Comment:
Should we change this default value in some way?  Or replace with an `example.com` email address?

Reply:
No issues, will change